docs/appendix/machine-specific-install.rst

```diff
@@ -1,11 +1,11 @@
 Machine-specific Install
 ========================
 
-NCAR HPC Cheyenne/Casper
+NCAR HPC Derecho/Casper
```

Comment:
I haven't used Derecho yet. Does anything need to be updated in this section?

Reply:
Not really, I've tried it and see no difference. Since Cheyenne does not exist anymore, it was just outdated to keep it. The only thing we should change (but it would be in the codebase) is the name of the environment.yaml